jaeger/cmd/install.go

```diff
@@ -41,7 +41,10 @@ func newCmdInstall() *cobra.Command {
 		Example: `  # Default install.
   linkerd jaeger install | kubectl apply -f -
   # Install Jaeger extension into a non-default namespace.
-  linkerd jaeger install --namespace custom | kubectl apply -f -`,
+  linkerd jaeger install --namespace custom | kubectl apply -f -
+  
+Configurable fields can be found at https://www.github.com/linkerd/linkerd2/tree/main/jaeger/charts/jaeger
```

Comment:
Better way to frame this? 🤔

Reply:
Maybe something like:

The installation can be configured by using the `--set` or `--values` flag.  A full list of configurable values can be found....